src/lib/query.ts

```diff
@@ -35,11 +35,16 @@ export class Query extends BaseQuery {
    * const result = await query.where("field_UID", QueryOperation.MATCHES, ["field1", "field2"]).find()
    * @returns {Query}
    */
-  where(fieldUid: string, queryOperation: QueryOperation, fields: string | string[] | number | number[]): Query {
+  where(
+    fieldUid: string, 
+    queryOperation: QueryOperation, 
+    fields: string | string[] | number | number[] | object | boolean,
+    additionalData?: object
```

Comment:
@abhinav-from-contentstack is this additionalData added for Taxonomy query?

Reply:
yes